src/test/java/org/mule/extension/file/integration/FileWriteTestCase.java

```diff
@@ -133,6 +135,7 @@ public void writeOnDirectoryPath() throws Exception {
 
   @Test
   public void writeOnDirectlyWithOutPermissions() throws Exception {
+    assumeFalse(IS_OS_WINDOWS);
```

Comment:
Why are we disabling this on Windows?

Reply:
Searching on the internet I noticed that there are several problems reported to the `JDK 7` with the use of` io.file` pq does not work as intended and it turns out that with `java.nio` it has` PosixFilePermission` is not supported for WINDOWS

This problem has already happened, here is the PR
mulesoft/mule-ee-distributions#389 (comment)